### DISCUSSION TITLE
Audiobookshelf integration into Music Player

### DISCUSSION THREAD

**Discussion by CunningSquirrel** [music-assistant/support#1478] [Feature requests and ideas]
Allow Music assistant to directly interface and stream audiobooks from self hosted Audiobookshelf server.

This would be a mind blowing addition to music assistant. It already has an API readily available. I just don’t have the skills to take advantage from it.

please consider making this a possibility for the community.

https://api.audiobookshelf.org/#introduction

**Comment by DesignsWithDavid** [comment]
This could also be achieved via a Subsonic API integration (https://github.com/advplyr/audiobookshelf/issues/399).

**Comment by OzGav** [comment]
Please add this to the audiobooks thread https://github.com/orgs/music-assistant/discussions/429

**Reply by jnbht** [reply]
But Audiobookshelf is its own selfhosted server. Not just an audiobooks provider.

**Comment by Tweedledeee** [comment]
+1 This would be really make my day!

**Comment by igbjnI05bF** [comment]
I would very much love this as well. Preferably with some way to keep track of listening positions if possible, so that you could continue listening to an audiobook/podcast from your car, walk in your house, and resume from musicassistant to snapcast for whole home audio

**Comment by StanDaMan0505** [comment]
That would be great.

**Comment by nikwilliamson** [comment]
Throwing my support in as well!
